app/src/evaluation/literalai_exporter.py

```diff
@@ -56,11 +60,13 @@ def to_csv_dict(self) -> dict[str, str]:
             "Agency ID": self.agency_id,
             "Session ID": self.session_id,
             "Program": self.program if self.program else "",
-            "Citation Links": self.citation_links if self.citation_links else "",
-            "Citation Sources": self.citation_sources if self.citation_sources else "",
+            "Citation Links": "\n".join(self.citation_links) if self.citation_links else "",
+            "Citation Sources": "\n".join(self.citation_sources) if self.citation_sources else "",
             "Has Chat History": str(self.has_chat_history),
             "Thread ID": self.thread_id,
             "Timestamp": self.timestamp,
+            "Feedback Scores": self.scores[0] if len(self.scores) == 1 else self.scores or "",
+            "Feedback Comments": self.comments[0] if len(self.comments) == 1 else self.comments or "",
```

Comment:
Crux of PR. 
Also if there's only zero or one item (99%+ of the time), don't format the items as a list (which uses brackets) -- just encode the single item as a string.